scripts/script.js

```diff
@@ -0,0 +1,155 @@
+const apiKey= '9853aca6055d1f5c122be191411d006b';
```

Comment:
```suggestion
const apiKey= process.env.API_Key;
```

Please use an env file to include the API_Key.
Also, don't forget to invalidate this API Key.